iroh-gateway/src/metrics.rs

```diff
@@ -1,8 +1,10 @@
 use iroh_metrics::config::Config as MetricsConfig;
 
+use crate::VERSION;
+
 pub fn metrics_config_with_compile_time_info(cfg: MetricsConfig) -> MetricsConfig {
     // compile time configuration
-    cfg.with_service_name(env!("CARGO_PKG_NAME").to_string())
```

Comment:
name and version are sth different 😅

Reply:
lmao of course that's what you meant, I just replaced the wrong line 🤦‍♀️ 
well, this is why we have review

Reply:
Okay, so I'm going to replace the `VERSION` to be the git version, to be used in the metrics files and the rpc `version` and `watch` calls

Reply:
hmmm.... the git version looks very bad on `iroh status -w` output

I think this may need further discussion. I'm going to file an issue like you originally asked.